### ISSUE TITLE
required service name and namespace in istio-proxy 

### ISSUE THREAD

**Issue by amankumarcs** [istio/istio#48048]
i have one requirement. basically below is the istio proxy logs 
`
{
  "attributes": {
    "destination": {
      "address": {
        "socketAddress": {
          "address": "192.168.7.167",
          "portValue": 9080
        }
      }
    },
    "metadataContext": {},
    "request": {
      "http": {
        "headers": {
          ":authority": "details:9080",
          ":method": "GET",
          ":path": "/details/0",
          ":scheme": "http",
          "accept": "*/*",
          "accept-encoding": "gzip, deflate",
          "user-agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36",
          "x-b3-parentspanid": "286e2e12989b14b4",
          "x-b3-sampled": "1",
          "x-b3-spanid": "8cf81d78cf0be544",
          "x-b3-traceid": "e3498612dae0df239090c0bedefd6f23",
          "x-envoy-attempt-count": "3",
          "x-forwarded-client-cert": "By=spiffe://cluster.local/ns/ct-next-gen/sa/nextgen-details;Hash=8ad25e8914f05a2034c35c2c05c2d3a92e1ae78823c32e2b185a572a9b20e67a;Subject=\"\";URI=spiffe://cluster.local/ns/ct-next-gen/sa/nextgen-productpage",
          "x-forwarded-proto": "http",
          "x-request-id": "16c124f6-eebf-96fe-81b5-5829e42a63b9"
        },
        "host": "details:9080",
        "id": "17203786893604877083",
        "method": "GET",
        "path": "/details/0",
        "protocol": "HTTP/1.1",
        "scheme": "http"
      },
      "time": "2022-12-19T10:17:20.642533Z"
    },
    "source": {
      "address": {
        "socketAddress": {
          "address": "192.168.25.144",
          "portValue": 60838
        }
      }
    }
  },
  "parsed_body": null,
  "parsed_path": [
    "details",
    "0"
  ],
  "parsed_query": {},
  "truncated_body": false,
  "version": {
    "encoding": "protojson",
    "ext_authz": "v3"
  }
}
`

so basically inside attribute we have source and destination details with ip address and port . so when k8 serviceA is trying to access K8s serviceB then i need serviceA name and namespace in source attribute and serviceB name and namespace in destination attribute .
